README.md

```diff
@@ -2,7 +2,7 @@
 
 Forward all your logs to [Rapid7 InsightOps](https://www.rapid7.com/products/insightops/), like a breeze.
 
-![logentries dashboard](https://raw.githubusercontent.com/nearform/docker-logentries/master/dashboard.png)
+![logentries dashboard](https://raw.githubusercontent.com/rapid7/r7insight_docker/master/dashboard.png)
```

Comment:
Nice to have: Up-to-date Ops screenshot 😉

Reply:
Caught me out there 😃